sklearn/cluster/affinity_propagation_.py

```diff
@@ -197,7 +197,10 @@ class AffinityPropagation(BaseEstimator, ClusterMixin):
     Parameters
     ----------
     damping : float, optional, default: 0.5
-        Damping factor between 0.5 and 1.
+        Damping factor (between 0.5 and 1) damps the
```

Comment:
Definitely here; user guide says the same in math. Do what you would find helpful as a user.